definitions/windows.yaml

```diff
@@ -24,6 +24,31 @@ labels: [Logs]
 supported_os: [Windows]
 urls: ['http://www.forensicswiki.org/wiki/Windows_XML_Event_Log_(EVTX)']
 ---
+name: ApplicationRegistration
+doc: Application Registration (AppPath) registry keys.
+sources:
+- type: REGISTRY_KEY
+  attributes:
+    keys:
+    - 'HKEY_LOCAL_MACHINE\\Software\\Microsoft\\Windows\\CurrentVersion\\App Paths\\*'
+    - 'HKEY_LOCAL_MACHINE\\Software\\Wow6432Node\\Microsoft\\Windows\\CurrentVersion\\App Paths\\*'
+supported_os: [Windows]
+urls:
+  - 'https://github.com/keydet89/RegRipper2.8/blob/master/plugins/apppaths.pl'
+  - 'http://www.hexacorn.com/blog/2013/01/19/beyond-good-ol-run-key-part-3/'
+  - 'https://msdn.microsoft.com/en-us/library/windows/desktop/ee872121(v=vs.85).aspx'
+---
+name: APPX_PROCESSEnvironmentVariable
+doc: APPX_PROCESS environment variable.
```

Comment:
I can do this - I was trying to follow the style of the other doc strings, which seemed fairly terse.